src/js/form.js

```diff
@@ -1179,6 +1188,6 @@ Form.prototype.goToTarget = function( target ) {
  * @type {string}
  * @default
  */
-Form.requiredTransformerVersion = '1.42.0';
+Form.requiredTransformerVersion = 'https://github.com/eyelidlessness/enketo-transformer.git#1617062';
```

Comment:
@Martijnr I think you mentioned this as extra protection against a Transformer version mismatch? How often is it the case that you bump the dependency without also bumping this? Has it caught mistakes before?

I can't find this documented beyond [just being listed](https://enketo.github.io/enketo-core/Form.html#.requiredTransformerVersion). If it is helpful, I think it would be a good thing to add to the readme or something.

Reply:
I found this check a bit odd, I don't often see version checks like this because versions are pinned in `package.json`. I understand (especially from working on this task) that `enketo-core` is tightly coupled to `enketo-transformer` and depends on certain functionality, but I wonder if there might be a way to express that more explicitly, e.g. by checking for a certain set of explicit, named features that the transformer supports? Obviously out of scope for this PR, but may also help reduce churn here.

Reply:
Yikes.